Objects/listobject.c

```diff
@@ -2717,6 +2717,35 @@ list___init___impl(PyListObject *self, PyObject *iterable)
     return 0;
 }
 
+static PyObject *
+list_vectorcall(PyObject *type, PyObject * const*args,
+                size_t nargsf, PyObject *kwnames)
+{
+    if (kwnames && PyTuple_GET_SIZE(kwnames) != 0) {
```

Comment:
Since https://github.com/python/cpython/pull/18980 is merged.
We can replace the logic into

```c
if (!_PyArg_NoKwnames("list", kwnames)) {
    return NULL;
}
```